src/glide/docs/contributing.md

```diff
@@ -224,6 +190,17 @@ pnpm mach test glide/browser/base/content/test/config/dist/browser_include.js
 > [!NOTE]
 > The file has to be the `dist/$file.js` version, you cannot pass TypeScript files yet.
 
+> [!NOTE]
+> Tests sometimes fail with this message:
+>
+> ```
+> "FAIL uncaught exception - NotFoundError: Node.insertBefore:
+> Child to insert before is not a child of this node
+> at apply_mutations/<@chrome://glide/content/document-mirror.mjs:170:23"
```

Comment:
Would you mind checking what `Node` this is talking about? I think you should be able to identify it by running the tests with `--jsdebugger` and setting the "pause on uncaught errors" flag or a similar debug setup